pyproject.toml

```diff
@@ -1,3 +1,64 @@
+[tool.poetry]
+name = "pelorus"
+version = "0.1.0" # we are not using it yet
+description = "Automate the measurement of organizational behavior"
+authors = [
```

Comment:
Is this generated from `OWNERS` file? If not we have two places where we need to update authors, one for OpenShift PROW and second here. How we ensure those are in-sync?

Can we simply say:

authors = "OWNERS" ?

Reply:
No, it is not automatically

Yes, we can specify a string

Going to put `authors = ["Pelorus community"]`